bigbluebutton-html5/imports/ui/components/logout-confirmation/container.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import { createContainer } from 'meteor/react-meteor-data';
+import { makeCall } from '/imports/ui/services/api';
+import Auth from '/imports/ui/services/auth';
+import userListService from '../user-list/service';
+import LogoutConfirmation from './component';
+
+class LogoutConfirmationContainer extends Component {
+    
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return(
+      <LogoutConfirmation {...this.props} />
+    );
+  }
+}
+
+export default createContainer(() => {
+    
+  const isModerator = userListService.getCurrentUser().isModerator;
+  const endMeeting = () => makeCall('endMeeting', Auth.credentials);
```

Comment:
This should be placed in the service.js.

I don't even know what would happen if you place makeCall('endMeeting'..) call in the createContainer, since Meteor is supposed to periodically check the variables coming through createContainer. Does it work, or the meeting just randomly ends?

Also, you don't have to create variables before returning, you can return them right away, like:

```
return {
    isModerator: userListService.getCurrentUser().isModerator,
    endMeeting...
};
```